services/QuillLMS/client/app/bundles/Staff/components/activityForm/topics.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react'
 
-import TopicColumn from './topicColumn'
+import TopicGroup from './topicGroup'
+
+import '../../../../../../app/assets/stylesheets/cms/activities.scss'
```

Comment:
Let's import this into a style file rather than into the component itself - that isn't really a pattern we use elsewhere (arguably it's one we should move towards, but I think better to be consistent for now).